tests/plugins/passwd

```diff
@@ -1,3 +1,7 @@
 # Test password file
 user:$6$1sSXVMBVKMF$uDStq59YfiuFxVF1Gi/.i/Li7Vwf5iTwg8LovLKmCvM5FsRNJM.OPWHhXwI2.4AscLZXSPQVxpIlX6laUl9570
-test_user:$6$.c9f9sAzs5YXX2de$GSdOi3iFwHJRCIJn1W63muDFQAL29yoFmU/TXcwDB42F2BZg3zcN5uKBM0.1PjwdMpWHRydbhXWSc3uWKSmKr.
\ No newline at end of file
+test_user:$6$.c9f9sAzs5YXX2de$GSdOi3iFwHJRCIJn1W63muDFQAL29yoFmU/TXcwDB42F2BZg3zcN5uKBM0.1PjwdMpWHRydbhXWSc3uWKSmKr.
+# Password for these is "test"
```

Comment:
I think we should have different passwords for different test users - otherwise a bug where `user1` can authenticate with password of `user2` might not be catched.

While this MR is not about authentication, having it in the example data is a liability in my eyes :)